json_schema/type/biomaterial/specimen_from_organism.json

```
@@ -115,6 +115,13 @@
             "type": "object",
             "$ref": "module/process/purchased_reagents.json",
             "user_friendly": "Purchased specimen"
+        },
+        "organ_donor": {
```

```suggestion
        "transplanted_organ": {
```

And unless you have a policy of always appending fields to the schema, should this be inserted after organ_parts?

---

`transplanted_organ` might be interpreted as the specimen was collected from a transplanted organ (after successful transplantation) rather than an organ that was intended for transplantation (transplantation not done).

Thanks for the suggestion about the order. I will insert the field after the organ_part.

---

I see what you mean. I worry, though, that `organ_donor` will be misinterpreted to mean that the organism the specimen was taken from is an organ donor in general. There is also a potential "collision" of terms in that the donor schema is called donor_organism. I would stay away from "donor" altogether and use a term derived from "transplant". The description field is helpful but is not always available when users look at the metadata downstream, like in a JSON or TSV manifest generated by Azul.

Since this ultimately represents information about a process, i.e., *how* the specimen was collected, should this maybe be a field of the [collection_protocol](https://github.com/HumanCellAtlas/metadata-schema/blob/master/json_schema/type/protocol/biomaterial_collection/collection_protocol.json) entity?

---

I see your point, about the word *donor*. In that perspective I would suggest `transplant_organ`. 

About moving to collection_protocol, this field does not describe how the specimen was collected (this can be described by [this](http://www.ebi.ac.uk/efo/EFO_0009124) ontology, in `collection_protocol.method.text`).
This is about how the specimen was treated after the extraction because it was proposed for transplant. Similar fields that describe the process after the collection that affects the quality of the sample exist in the specimen level, like `specimen_from_organism.state_of_specimen.postmortem_interval`.

---

I like `transplant_organ`.